app/views/passengers/index.haml

```diff
@@ -48,7 +48,7 @@
     %b Showing All Passengers
 %p
 - if @passengers.present?
-  %table.row-border#passengers
+  %table{id: 'passengers', class: table_class}
```

Comment:
Since the ID is static, you can write it as `%table#passengers{class: table_class}`.